frontend/src/App.tsx

```diff
@@ -1,37 +1,38 @@
-import React, {useState} from 'react';
+import React from 'react';
 import './App.css';
 import {BrowserRouter, Route, Routes} from "react-router-dom";
 import SignUp from "./component/SignUp";
 import UserOverview from "./component/UserOverview";
-import {User} from "./model/User";
 import AddVisitCountry from "./component/AddVisitCountry";
 import useCountries from "./hook/useCountries";
 import WelcomeScreen from "./component/WelcomeScreen";
+import Login from "./component/Login";
+import useLoggedInUser from "./hook/useLoggedInUser";
+import ProtectedRoutes from "./component/ProtectedRoutes";
 
 
 function App() {
-    const [loggedInUser, setLoggedInUser] = useState<User>({id: "null", name: "loading", visitedCountries: []})
+
     const {countries} = useCountries()
+    const {loggedInTraveller, loginUser, username} = useLoggedInUser()
```

Comment:
Why would you receive a traveller if everything else in the line references a user?